casper/src/main/scala/coop/rchain/casper/protocol/CommUtil.scala

```diff
@@ -138,14 +138,14 @@ final class CommUtilOps[F[_]](
     sendToPeers(ForkChoiceTipRequest.toProto) >>
       Log[F].info(s"Requested fork tip from peers")
 
-  def requestApprovedBlock(
+  def requestFinalizedFringe(
       trimState: Boolean = true
   )(implicit m: Sync[F], r: RPConfAsk[F]): F[Unit] =
     for {
       maybeBootstrap <- RPConfAsk[F].reader(_.bootstrap)
       bootstrap      <- maybeBootstrap.liftTo(StandaloneNodeSendToBootstrapError)
-      msg            = ApprovedBlockRequest("", trimState).toProto
-      _              <- commUtil.sendWithRetry(ToPacket(msg), bootstrap, 10.seconds, "ApprovedBlockRequest")
+      msg            = FinalizedFringeRequest("", trimState).toProto
```

Comment:
I don't know what's the purpose of _identifier_ on ApprovedBlockRequest (FinalizedFringeRequest). It's filled with emoty string and should probably be removed.